mlir/lib/Conversion/ArithToEmitC/ArithToEmitC.cpp

```diff
@@ -22,6 +23,30 @@
 
 using namespace mlir;
 
+namespace {
+/// Implement the interface to convert Arith to EmitC.
+struct ArithToEmitCDialectInterface : public ConvertToEmitCPatternInterface {
+  using ConvertToEmitCPatternInterface::ConvertToEmitCPatternInterface;
+  void loadDependentDialects(MLIRContext *context) const final {
+    context->loadDialect<emitc::EmitCDialect>();
```

Comment:
The doc said that EmicDialect does not need to be loaded and this is for other dialects.

Reply:
Ping on this? Shall we remove this method?